README.rst

```diff
@@ -83,6 +82,62 @@ After running `aws-mfa`, your credentials file would read:
     aws_secret_access_key = <POPULATED_BY_AWS-MFA>
     aws_security_token = <POPULATED_BY_AWS-MFA>
 
+The default naming convention for the credential section can be overriden by using the ``--long-term-suffix`` and
+``--short-term-suffix`` command line arguments. For example, in a multi account scenario you can have one AWS account
```

Comment:
I'm not sure I follow. If you are in a multi-account scenario, your `long-term` credentials are going to be different for each account. So, you shouldn't be able to re-use one pair of `long-term` credentials on multiple accounts.

I sort of understand the need to customize the prefix. But below, you've just replaced the value `default` with `myorganization`, so there is no functional difference. Try taking a look at: http://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html for more details on named profiles.